singlestoredb/tests/test_basics.py

```diff
@@ -977,6 +977,11 @@ def _test_MySQLdb(self):
         for a, b in zip(s2_out, mydb_out):
             assert a == b, (a, b)
 
+    def test_long_string(self):
```

Comment:
If you use 49 instead of 48, there could be a test for parsing doubles as well.
```
    def test_double_string(self):
         string = 'a' * 49
         self.cur.execute(f"SELECT 1.2 :> double AS column_1, '{string}' AS column_2")
         self.assertEqual((1.2, string), self.cur.fetchone())
```